sailthru_content/services/sailthru_translation_service.py

```diff
@@ -97,6 +97,16 @@ def _get_tags_from_property(self, source_array, key, tag_prefix):
 
         return tags
 
+    def _return_sku(self, seats):
+        """ Return dict of sku for a course run. """
+        sku = {}
```

Comment:
Every seat type has a SKU. Why the limitations?

`return {seat['type']: seat['sku'] for seat in seats}`